server/submissionWorkflow/api.ts

```diff
@@ -0,0 +1,66 @@
+import app, { wrap } from 'server/server';
+import { ForbiddenError } from 'server/utils/errors';
+
+import { getPermissions } from './permissions';
+import {
+	createSubmissionWorkflow,
+	updateSubmissionWorkflow,
+	destroySubmissionWorkFlow,
+} from './queries';
+
+const getRequestIds = (req) => {
+	const user = req.user || {};
+	return {
+		userId: user.id,
+		communityId: req.body.communityId,
```

Comment:
3️⃣ You should be able to eliminate `communityId` from all of your API code — providing a `collectionId` should be enough to uniquely identify a Collection.

Reply:
Please do give this a shot (and let me know if it's not working, that's indicative of a problem with `getScope` that we need to fix)